calculator.py

```diff
@@ -0,0 +1,11 @@
+user_input = input("Type in your yearly salary = ")
+

```

Comment:
Please head a header mentioning author name and file description

Reply:
Can I go in and edit the code from github, or do I need to upload a new file with the changes??

Reply:
done